assets/sass/_api-nodejs.scss

```diff
@@ -42,7 +42,9 @@ ul.api {
         @apply truncate mt-0;
 
         a {
-            @apply btn bg-transparent inline-block p-1;
+            @apply btn truncate bg-transparent inline-block p-1;
```

Comment:
If we're going to truncate these, we should make sure they have `title` tags also, so you can see what they are. (I'm usually not a fan of truncating because it obscures important info, but when it's necessary, it should always be possible to figure out what's been lopped off.)

Reply:
> If we're going to truncate these, we should make sure they have title tags also, so you can see what they are

Yep I did this.

> I'm usually not a fan of truncating because it obscures important info, but when it's necessary, it should always be possible to figure out what's been lopped off.

They get cut-off today anyway if the name is too long. This just ensures that we show an ellipsis, as well as show a tooltip using the `title` attribute. Admittedly, latter is a change I have made only to the resource docs templates and not to the API docs ones.